website/community/security/CVE-2016-8737.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: website-normal
+title: "CVE-2016-8737: Cross-site request forgery vulnerability in Apache Brooklyn"
+---
+
+## Severity
+Major
+
+## Vendor
+The Apache Software Foundation
+
+## Versions Affected
+Apache Brooklyn 0.9.0 and all prior versions
+
+## Description
+Apache Brooklyn's REST server is vulnerable to cross-site request forgery (CSRF), which could permit a malicious web site to produce a link which, if clicked whilst a user is logged in to Brooklyn, would cause the server to execute the attacker's commands as the user.  There is known to be a proof-of-concept exploit using this vulnerability.
+
+## Solution
+Upgrade to Apache Brooklyn 0.10.0. This includes commit [1] adding opt-in CSRF protection server-side and commit [2] where the JS client opts-in.  
```

Comment:
I would make the reference numbers links to either the source article or the references. Perhaps make Apache Booklyn 0.10.0 a link to the download page?